docs/content/Coding Milestones/PoC2023q1/commands.md

```diff
@@ -742,8 +762,9 @@ the list of things that went into the WEC.
 
 The following command is a combination of `kubectl kubestellar
 ensure-location` and `kubectl kubestellar prep-for-syncer`, and takes
-the union of their command line flags and arguments.  Upon completion,
-the kcp current workspace will be what it was at the start.
+the union of their command line flags and arguments and reads the
+union of their environment variable read sets.  Upon completion, the
+kcp current workspace will be what it was at the start.
```

Comment:
This is KCP only thing. We no longer have the notion of navigation. There is no "current" space

Reply:
fixed